tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html

```diff
@@ -412,24 +426,16 @@
       <br style="clear: both">
       <div>Devices included in stats:</div>
       <div class="deviceList">
-        <table>
-        <template is="dom-repeat" items="[[_getDevices(devicesForStats)]]">
-          <tr>
-            <td>
-              <input type="checkbox" value$="[[item.device]]" checked$="[[item.used]]" on-click="_deviceCheckboxClicked"/>
-            </td>
-            <td>
-              <div>
-                <span>[[item.suffix]]</span>
-                <template is="dom-if" if="[[item.ignoredMsg]]">
-                  <paper-icon-button icon="help" class="help-icon"></paper-icon-button>
-                  <paper-tooltip position="right" animation-delay="0">[[item.ignoredMsg]]</paper-tooltip>
-                </template>
-              </div>
-            </td>
-          </tr>
+        <template is="dom-repeat" items="[[_currentDevices]]">
+          <div id="devices-checkbox" class="color-legend-row">
```

Comment:
Within a web page, each ID can be used once. Could we make `devices-checkbox` a class name alongside `color-legend-row`?

Reply:
Nice.  Done.